.github/workflows/build-publish.yaml

```diff
@@ -34,34 +34,18 @@ jobs:
   build:
     runs-on: ubuntu-latest
     needs: prepare
-    environment:
-        name: cloudops
```

Comment:
We will potentially need to move the `DOCKERHUB_USERNAME` and `DOCKERHUB_TOKEN` secrets from Environment Secrets to Repository Secrets for publishes to work.

Reply:
I think that might require some assistance from @kkleemola

Reply:
> I think that might require some assistance from @kkleemola

We were chatting about this a bit this morning, but I wanted to get input from you (@bsieber-mozilla) and @leplatrem before we moved forward with this